pkgs/top-level/python-packages.nix

```diff
@@ -4225,6 +4225,24 @@ let
     };
   };
 
+  wtforms = buildPythonPackage rec {
+    version = "2.0.2";
+    name = "wtforms-${version}";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/W/WTForms/WTForms-${version}.zip";
+      md5 = "613cf723ab40537705bec02733c78d95";
+    };
+
+    propagatedBuildInputs = with self; [ ordereddict Babel ];
+
+    meta = {
+      homepage = https://github.com/wtforms/wtforms;
+      description = "A flexible forms validation and rendering library for Python.";
+      license = "BSD";
```

Comment:
This could be more precise, e.g., `licenses.bsd3`. Also, there is no need for a dot at the end of the `description`. Thanks.